### PR TITLE
Configure index product on category products update

### PR DESCRIPTION
**Pull request by JanPetr**

### CODE REVIEW

File: code/etc/system.xml [hunk 1]
```diff
@@ -708,6 +708,16 @@
...
                         </remove_disabled_products_from_index>
+                        <index_product_on_category_products_update translate="label comment">
+                            <label>Update product on category products update</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>70</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment>This feature enables the extension to automatically update the product index when a product is added to or removed from a category via the "Manage Categories" menu.</comment>
```
Comment (maxiloc) [inline code/etc/system.xml]: Maybe something like : "By default ....., Setting it to false will disable that behavior. This is useful for ...." would be better

**Comment by maxiloc** [comment]
👍

File: code/etc/system.xml [hunk 2]
```diff
@@ -708,6 +708,22 @@
...
+                            <sort_order>70</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment>
+                                <![CDATA[
+                                This feature enables the extension to automatically update the product index when a product is added to or removed from a category via the "Manage Categories" menu.
+                                By default the feature is turned on. <br />
+                                Setting it to "No" will disable its behavior. It is useful when you hava a lot of products in catalog and saving the category will synchronize them all with Algolia even though there is no real change.
+                                ]]>
```
Comment (JanPetr) [inline code/etc/system.xml]: Better like that @maxiloc? ✏️

Comment (maxiloc) [inline code/etc/system.xml]: yes way better

Comment (JanPetr) [inline code/etc/system.xml]: Will work on my documentary skills 🎓 :-D

**Pull request merged**